neuror/main.py

```diff
@@ -248,26 +249,24 @@ def _max_y_dendritic_cylindrical_extent(neuron):
 class Repair(object):
     '''The repair class'''
 
-    def __init__(self, inputfile, axons=None, seed=0, plane=None):
+    def __init__(self, inputfile, axons=None, seed=0, cut_leaves_coordinates=None):
         '''Repair the input morphology
 
-        Note: based on https://bbpcode.epfl.ch/browse/code/platform/BlueRepairSDK/tree/BlueRepairSDK/src/repair.cpp#n469  # noqa, pylint: disable=line-too-long
+        Note: based on
+        https://bbpcode.epfl.ch/browse/code/platform/BlueRepairSDK/tree/BlueRepairSDK/src/repair.cpp#n469
         '''
         np.random.seed(seed)
-        if plane is None:
-            L.info('No cut plane specified. Calling CutPlane.find...')
-            cut_plane = CutPlane.find(inputfile, bin_width=15)
-            L.info('Found cut plane: %s', cut_plane)
-        else:
-            if isinstance(plane, CutPlane):
-                cut_plane = plane
-            else:
-                cut_plane = CutPlane.from_json(plane)
 
         self.inputfile = inputfile
         self.axon_donors = axons or list()
         self.donated_intact_axon_sections = list()
-        self.cut_leaves = cut_plane.cut_leaves_coordinates
+
+        if cut_leaves_coordinates is None:
+            L.info('No cut plane specified. Calling CutPlane.find...')
```

Comment:
still the same info than before ?

Reply:
Yes, we still do the same thing if the cut points were not passed by the user.